test/system/helpers.bash

```diff
@@ -1,5 +1,11 @@
 # -*- bash -*-
 
+# Ignore cgroups-v1 warning in cirrus
+if [[ ! -z "$CIRRUS_CI" && $(mount | grep -q cgroup2) -ne 0 ]]; then
+    # Actual value of this envvar doesn't matter
+    IGNORE_CGROUPSV1_WARNING=1
```

Comment:
as env var I assume this needs a `export` otherwise it is not passed to child processes

Reply:
ah yup